cunumeric/module.py

```diff
@@ -2448,6 +2448,113 @@ def flip(m: ndarray, axis: Optional[NdShapeLike] = None) -> ndarray:
 # Generating index arrays
 
 
+@add_boilerplate("arr", "mask", "vals")
+def place(arr: ndarray, mask: ndarray, vals: ndarray) -> None:
+    """
+    Change elements of an array based on conditional and input values.
+
+    Parameters
+    ----------
+    arr : array_like
+        Array to put data into.
+    mask : array_like
+        Boolean mask array. Must have the same size as `arr`.
```

Comment:
I think you can remove `Boolean` from documentation since numPy and your implementation works with other types